pandas/io/formats/css.py

```diff
@@ -27,12 +27,46 @@ def expand(self, prop, value: str):
     return expand
 
 
+def _border_expander(side: str = ""):
+    if side != "":
+        side = f"-{side}"
+
+    def expand(self, prop, value: str):
+        tokens = value.split()
+        if len(tokens) == 0 or len(tokens) > 3:
+            warnings.warn(
+                f'Too many tokens provided to "{prop}" (expected 1-3)', CSSWarning
```

Comment:
tests on this? why is this not an exception?

Reply:
CSS errors are silently ignored in browsers. I was agnostic about whether we should ignore, warn or raise on errors like this (what if CSS language developed in future and we were still raising), in the end I went with ignore, OP went with warn.

Reply:
Tested in `test_css_parse_invalid`